src/Admin/Filter/Types/DateRangeFilterType.php

```diff
@@ -24,10 +25,12 @@ public function execute(FilterItem $filterItem, Builder $builder): void
         $max = $this->getRangeValue(static::KEY_MAX);
 
         if ($min) {
+            $min = Carbon::parse($min)->startOfDay()->toDateTimeString();
```

Comment:
Why does this needs to be parsed when date format always is `Y-m-d`?
Also  startOfDay/endOfDay is irrelevent since whereDate ignores any timestamps